### PR TITLE
[JOLINT-65] title section for inclusion page

### PR DESCRIPTION
**Pull request by xxrobone**

### CODE REVIEW

File: app/(info-page)/inclusion/page.tsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
+import TitleSectionInclusion from '@/app/(info-page)/_components/TitleSectionInclusion'
+
 const Inclusion = () => {
-  return <div>inclusion</div>
+  return (
+    <div className='min-h-screen relative px-20 py-28'>
```
Comment (ell-ska) [inline app/(info-page)/inclusion/page.tsx:5]: there has been an update with a general main layout (in layout.tsx) that will clash with the inline padding you have here. i would suggest to merge in main to get the new updates. check out the page.tsx in faq where ida did a similar section if you want to see her way of adapting to the general layout

**Review by ell-ska — CHANGES_REQUESTED** [review]

File: app/(info-page)/inclusion/page.tsx (hunk 1 above)
Comment (xxrobone) [reply app/(info-page)/inclusion/page.tsx:5]: I've done the changes, I will wait for main to be fixed and then pull it down to my local, and check all changes.

**Review by ell-ska — APPROVED** [review]

**Review by ell-ska — APPROVED** [review]

**Comment by idaakermark** [comment]
Testing on dev: There is too much padding on the title which needs to be fixed!

**Comment by idaakermark** [comment]
testing on dev: need to change the font-color. Other than that, everything looks good!

**Comment by idaakermark** [comment]
testing on dev: Looks good, ready for production.

**Pull request merged**